_component_design/switches.md

```diff
@@ -106,23 +193,96 @@ Best practice form design does not allow a switch to exist in an error state, so
 
 <section class="static-section" markdown="1">
 
-## Context
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Disabled
+
+A user can't select a switch in the disabled state. In the disabled state, the switch is locked and cannot be changed.
```

Comment:
kill the first sentence here. the 2nd sentence is sufficient

Reply:
Done.